hazelcast/src/main/java/com/hazelcast/jet/core/Processor.java

```diff
@@ -87,12 +88,13 @@
  *
  * <h3>How the methods are called</h3>
  * <p>
- * Besides {@link #init}, {@link #close} and {@link #isCooperative} the methods
- * are called in a tight loop with a possibly short back-off if the method does
- * no work. "No work" is defined as adding nothing to outbox and taking nothing
- * from inbox. If you do heavy work on each call (such as querying a remote
- * service), you can do additional back-off: use {@code sleep} in a
- * non-cooperative processor or do nothing if sufficient time didn't elapse.
+ * Besides {@link #init}, {@link #close}, {@link #isCooperative} and {@link
+ * #closeIsCooperative()} the methods are called in a tight loop with a possibly
+ * short back-off if the method does no work. "No work" is defined as adding
+ * nothing to outbox and taking nothing from inbox. If you do heavy work on each
+ * call (such as querying a remote service), you can do additional back-off: use
+ * {@code sleep} in a non-cooperative processor or do nothing if sufficient time
+ * didn't elapse.
```

Comment:
```suggestion
 * Except for {@link #init}, {@link #close}, {@link #isCooperative} and {@link
 * #closeIsCooperative()}, the methods are called in a tight loop with a
 * possibly short back-off if the method does no work. "No work" is defined as
 * adding nothing to outbox and taking nothing from inbox. If you do heavy work
 * on each call (such as querying a remote service), you can do additional
 * back-off: use {@code sleep} in a non-cooperative processor or do nothing if
 * sufficient time didn't elapse.
```
"Besides" means "in addition to." Instead of it, "except for" should be used and the clause should end with a comma.